libcudacxx/.upstream-tests/utils/libcudacxx/test/format.py

```diff
@@ -210,14 +217,18 @@ def _evaluate_pass_test(self, test, tmpBase, lit_config,
                                                       env)
                 report = "Compiled With: '%s'\n" % ' '.join(compile_cmd)
                 report += libcudacxx.util.makeReport(cmd, out, err, rc)
-                if rc == 0:
+                result_expected = (rc == 0) == run_should_pass
```

Comment:
One day I will rewrite this as a switch over the number of possibilities but that day is not today